test/Microsoft.Health.Fhir.Shared.Tests.E2E/Rest/Search/BasicSearchTests.cs

```diff
@@ -1043,6 +1043,29 @@ public async Task GivenSetOfChunkyResources_WhenIteratingOverThemWithSort_ThenAl
             Assert.Equal(Enumerable.Range(0, n), values.OrderBy(x => x));
         }
 
+        [Fact]
+        public async Task GivenASearchRequestWithParameter_TextAndLenientHandling_WhenHandled_ReturnsSearchResults()
```

Comment:
Nit: The _ between Parameter and Text should be removed for consistency. Also in the test name below this one.

Reply:
> Nit: The _ between Parameter and Text should be removed for consistency. Also in the test name below this one.

Done.